src/components/views/rooms/wysiwyg_composer/hooks/useSuggestion.ts

```diff
@@ -144,7 +146,7 @@ export function processMention(
     attributes: Attributes, // these will be used when formatting the link as a pill
```

Comment:
Probably not for this change, but where do these attributes come from? Should this be a `Map` to avoid prototype injection?

Reply:
These attributes come from the wysiwyg repo. I think the naming could be improved to clarify that in a subsequent release and I'll also open an issue about how we manage these attributes, whether we should move to a `Map` (as this would require changes in both RTE and element web).

Reply:
Sounds good.